aip/0192.md

```diff
@@ -102,9 +102,15 @@ in `(--` and `--)`.
 Non-public links, internal implementation notes (such as `TODO` and `FIXME`
 directives), and other such material **must** be marked as internal.
 
+**Note:** Comments **should not** use both a leading and trailing comment to
```

Comment:
Shouldn't we just say "no trailing comments" ? Wouldn't that be easier? I feel like trailing comments are only good for short post-field thoughts (e.g., `int32 id = 1; // This should always be blank`) which can just as easily be done as a leading comment and should be more future proof (when it gets long, you know exactly what to do)

Reply:
That seems reasonable.

Reply:
This doesn't look to be resolved -- are we going to change that?

Reply:
Whups! Done.